src/App.tsx

```diff
@@ -1,35 +1,46 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import React, { useState } from "react";
+import { Form } from "./assets/components/Form/Form.tsx";
+import { Cards } from "./assets/components/Cards/Cards.tsx";
+import "./App.css";
 
-function App() {
-  const [count, setCount] = useState(0)
+interface Card {
+  id: string;
+  name: string;
+  difficulty: string;
+  price: number;
+}
+
+const App: React.FC = () => {
+  const [cards, setCards] = useState<Card[]>([]);
+
+  const handleFormSubmit = (newCard: {
+    id: string;
+    name: string;
+    difficulty: string;
+    price: number;
+  }) => {
```

Comment:
you could simply use here already created type like ` const handleFormSubmit = (newCard: Card) => {` to avoid code duplication and actual type safety